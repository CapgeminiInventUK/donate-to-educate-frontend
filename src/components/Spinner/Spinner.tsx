import { FC } from 'react';
import styles from './Spinner.module.scss';
import spinner from './spinner.gif';

export const Spinner: FC = () => {
  return (
    <div className={styles.container}>
      <img src={spinner} alt="loading..." />
    </div>
  );
};

export default Spinner;
