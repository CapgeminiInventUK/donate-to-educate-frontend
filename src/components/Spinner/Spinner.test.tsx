import { create } from 'react-test-renderer';

import Spinner from './Spinner';

it('renders correctly', () => {
  const tree = create(<Spinner />).toJSON();
  expect(tree).toMatchSnapshot();
});
