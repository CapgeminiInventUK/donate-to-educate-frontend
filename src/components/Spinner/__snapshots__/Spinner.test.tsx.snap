// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders correctly 1`] = `
<div
  className="_container_d122df"
>
  <img
    alt="loading..."
    src="/src/components/Spinner/spinner.gif"
  />
</div>
`;
